### DISCUSSION TITLE
Changing username

### DISCUSSION THREAD

**Discussion by adamhele** [community/community#102515] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I named my GitHub account @chickenboyisthebest when I started because I just wanted to keep all of my code online easily. Now, however, I want to put wildlife things on and thus change my username. Will it delete the code on my GitHub repositories including the GitHub pages one.

**Comment by ghost** [comment]
To change your GitHub username, log in, go to your profile, and access the account settings. Find the "Username" field, click "Edit," and enter your new desired username. Confirm the change by entering your password and possibly verifying through an email link.

This works I gueas

**Reply by adamhele** [reply]
Will it delete anything? Thank you so much in advance!

**Reply by ghost** [reply]
It didn't do anything like that to me

**Reply by adamhele** [reply]
Thank You.

**Reply by ghost** [reply]
I referred this vid while doing it

**Reply by ghost** [reply]
[Tutorial](https://youtu.be/wlDzXCwhDeI?si=pkvToVQDtaF4cSbh)

**Comment by adamhele** [comment]
Thank You

**Reply by ghost** [reply]
Your welcome :)
